src/poolboy.erl

```diff
@@ -143,18 +141,19 @@ init([{strategy, lifo} | Rest], WorkerArgs, State) ->
 init([{strategy, fifo} | Rest], WorkerArgs, State) ->
     init(Rest, WorkerArgs, State#state{strategy = fifo});
 init([{overflow_ttl, OverflowTtl} | Rest], WorkerArgs, State) when is_integer(OverflowTtl) ->
-    init(Rest, WorkerArgs, State#state{overflow_ttl = OverflowTtl});
+    TimerTTL = min(OverflowTtl, ?RIP_PERIOD),
```

Comment:
Can we make this item configurable?

Also can we make the case and naming consistent eg. ReapTime TTL refers to Time To Live the timer that takes care of how frequently we reap workers doesn't have a Time To Live, the overflow workers do.